.devcontainer/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM python:3.8-slim
+
+ENV POETRY_VERSION=1.6.1 \
+    POETRY_NO_INTERACTION=1 \
+    POETRY_VIRTUALENVS_IN_PROJECT=false \
+    POETRY_VIRTUALENVS_CREATE=false \
+    DEBIAN_FRONTEND=noninteractive \
+    DISPLAY=:99
+
+# Install packages
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    sudo git build-essential chromium chromium-driver \
+    && rm -rf /var/lib/apt/lists/*
+
+# Create remote user
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+RUN groupadd --gid $USER_GID $USERNAME \
+    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+    && echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME \
+    && chmod 0440 /etc/sudoers.d/$USERNAME
+
+ENV PATH="/home/${USERNAME}/.local/bin:${PATH}"
+
+# Install nicegui
+RUN pip install -U pip
```

Comment:
Duplicate of next line, generally better to combine Run commands as far as possible

Reply:
Oh, sorry. I was not very attentive. I'll revert my commit!